.travis.yml

```diff
@@ -0,0 +1,68 @@
+sudo: false
+
+language: php
+
+php:
+    - 7.2
+    - 7.3
+    - nightly
+
+before_install:
+    - &Xdebug phpenv config-rm xdebug.ini || true
+
+install:
+    - travis_retry composer install --classmap-authoritative --no-suggest --prefer-dist
+
+script:
+    - vendor/bin/phpunit
+
+jobs:
+    include:
+
+        - stage: Test
+          name: Symfony 4.0
+          before_install:
+              - *Xdebug
+              - composer require --dev --no-update symfony/symfony=4.0.*
```

Comment:
Trying to work out if there's a better way to do this. This will break if we ever use `^4.1` etc, which is likely given `4.0` is no longer supported. Though maybe shouldn't then test for it...

Reply:
Want to avoid maintaining a list of components here.

Reply:
Maybe checking the `3` and `4` branches is enough?

Reply:
Not going to worry for now, will leave this the same as elsewhere.